docs/telescope.txt

```diff
@@ -0,0 +1,34 @@
+================================================================================
+                                                                *telescope.nvim*
+
+Telescope.nvim is a plugin for fuzzy finding and neovim. It helps you search
```

Comment:
🤣🤣🤣 god damn this so unfair to telescope . I will suggest a better one later. or in new pr